romancal/datamodels/schemas/core.schema.yaml

```diff
@@ -1,6 +1,5 @@
 %YAML 1.1
 ---
-$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
```

Comment:
Opinions may vary, but I don't think we should remove this entirely.  We don't need the FITS validators defined in the jwst metaschema but I think we should still specify _some_ metaschema.  Since we're not using the ndarray validators here the base YAML metaschema is appropriate:

```yaml
schema: http://stsci.edu/schemas/yaml-schema/draft-01
```

or we could just use the ASDF metaschema everywhere so we don't have to think about whether or not the array features are necessary:

```yaml
$schema: http://stsci.edu/schemas/asdf/asdf-schema-1.0.0
```

Incidentally, the quotes around URI values aren't necessary.   There aren't any characters in there that YAML parsers find confusing.

Reply:
Done.